website/src/pages/roadmap.md

```diff
@@ -4,69 +4,55 @@ last_modified_at: 2019-12-30T15:59:57-04:00
-|Indexed columns support & elimination of partitioning |0.11|[HUDI-512](https://issues.apache.org/jira/browse/HUDI-512)|
-|Record-level index to speed up uuid based upserts/deletes |0.12|[HUDI-53](https://issues.apache.org/jira/browse/HUDI-53)|
-|Eager conflict detection for Optimistic Concurrency Control |0.12|[HUDI-1575](https://issues.apache.org/jira/browse/HUDI-1575)|
-|Indexed timeline and infinite retention of versions |0.12|RFC coming soon|
-|Improvements to streaming read and full CDC data model support |0.12| [HUDI-2749](https://issues.apache.org/jira/browse/HUDI-2749), RFC coming soon|
-|Consistent hashing based file distribution over storage to overcome throttling issues for very large tables |0.12|RFC published soon|
-|Lock free concurrency control |0.12 -> 1.0.0|[HUDI-3187](https://issues.apache.org/jira/browse/HUDI-3187)|
-|Non-blocking/Lock-free updates during clustering |0.12 -> 1.0.0|[HUDI-1042](https://issues.apache.org/jira/browse/HUDI-1042)|
-|Time Travel updates, deletes |0.12 -> 1.0.0 ||
-|General purpose support for multi-table transactions |0.12 -> 1.0.0||
+| Metaserver for all metadata | 0.13 | [Strawman design](https://docs.google.com/presentation/d/1QBgLw11TM2Qf1KUESofGrQDb63EuggNCpPaxc82Kldo/edit#slide=id.gf7e0551254_0_5), [RFC-36](https://github.com/apache/hudi/pull/4718), [HUDI-3345](https://issues.apache.org/jira/browse/HUDI-3345) |
+| Supervised table service planning and execution | 0.13 | [RFC-43](https://github.com/apache/hudi/pull/4309), [HUDI-4147](https://issues.apache.org/jira/browse/HUDI-4147) |
+| Support of Change Data Capture (CDC) with Hudi change logs | 0.13 | [RFC-51](https://github.com/apache/hudi/blob/master/rfc/rfc-51/rfc-51.md), [HUDI-3478](https://issues.apache.org/jira/browse/HUDI-3478) |
+| Record-level index to speed up UUID-based upserts and deletes | 0.13 | [RFC-08](https://cwiki.apache.org/confluence/display/HUDI/RFC-08++Record+level+indexing+mechanisms+for+Hudi+datasets), [HUDI-53](https://issues.apache.org/jira/browse/HUDI-53) |
+| Consistent hashing index for dynamic buckets | 0.13 | [RFC-42](https://github.com/apache/hudi/blob/master/rfc/rfc-42/rfc-42.md), [HUDI-3000](https://issues.apache.org/jira/browse/HUDI-3000) |
+| Secondary index to improve query performance | 0.13 | [RFC-52](https://github.com/apache/hudi/pull/5370), [HUDI-3907](https://issues.apache.org/jira/browse/HUDI-3907) |
+| Reducing write amplification with Log Compaction in MOR | 0.13 | [RFC-48](https://github.com/apache/hudi/pull/5041), [HUDI-3580](https://issues.apache.org/jira/browse/HUDI-3580) |
+| Eager conflict detection for Optimistic Concurrency Control | 0.13 | [RFC-56](https://github.com/apache/hudi/pull/6003), [HUDI-1575](https://issues.apache.org/jira/browse/HUDI-1575) |
+| Efficient bootstrap and migration of existing non-Hudi dataset | 0.13 | [HUDI-1265](https://issues.apache.org/jira/browse/HUDI-1265) |
```

Comment:
bootstrap/migration should belong to "platform services" ?

Reply:
Bootstrap and migration involve transaction-layer changes so I put it here.